azure-pipelines.release-fluentui.yml

```diff
@@ -1,74 +1,220 @@
-      storage: fluentsite
-      ContainerName: '$web'
-      BlobPrefix: '$(deployBasePath)'
-      # uncomment to deploy to prod
-      # storage: fluentsite
-      # BlobPrefix: ''
-# No NPM release in CI for now
-#  - script: |
-#      yarn test
-#    displayName: yarn test
-#
-#  - script: |
-#      yarn lint
-#    displayName: yarn lint
-#
-#  - script: |
-#      yarn release:fluentui:$(bumpType)
-#    displayName: yarn release:fluentui:$(bumpType)
+    value: true
+  - name: nightlyRelease
+    value: true
+  # below are variables configurable at queue time, so they cannot appear in yml file
+  # - name: publishDocsiteOnly # set it to true to only run Job_2
+  #   value: false
+  # - name: publishOfficial # set it to true to publish package to npm, and publish docsite for official releases
+  #   value: false
+  # - name: version # version to publish to npm; only make sense when publishOfficial=true
+  #   value: canary # value can only be canary/patch/minor
+
+jobs:
+  - job: Job_1
+    displayName: Build and Release Fluent Packages
+    # skip this job if publishDocsiteOnly is true
+    condition: and(succeeded(), eq(variables.publishDocsiteOnly, false))
+    pool:
+      name: Self Host Ubuntu
+
+    steps:
+      - checkout: self
+
+      - task: CmdLine@2
+        displayName: Authenticate Git credentials
+        inputs:
+          script: >-
+            git config user.name "Fluent UI Build"
+
+            git config user.email "fluentui-internal@service.microsoft.com"
+
+            git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/microsoft/fluentui.git
+
+      - task: CmdLine@2
+        displayName: Checkout branch
+        inputs:
+          script: >-
+            git checkout $(Build.SourceBranchName)
+
+            git pull
+
+      - task: NodeTool@0
+        displayName: Install Node.js
+        inputs:
+          versionSpec: 12.x
+
+      - task: YarnInstaller@0
+        displayName: Use Yarn 1.19.x
+        inputs:
+          versionSpec: 1.19.x
+          checkLatest: true
+
+      - task: Bash@3
+        displayName: Yarn
+        inputs:
+          filePath: yarn-ci.sh
+
+      - task: CmdLine@2
+        displayName: yarn buildci
+        inputs:
+          script: yarn buildci
+
+      - task: CmdLine@2
+        displayName: '[Nightly] Pack fluentui packages'
+        condition: and(succeeded(), eq(variables.publishOfficial, false))
+        inputs:
+          script: >-
+            today=`date +%Y-%m-%d`
+
+            echo date today $today
+
+            echo "##vso[task.setvariable variable=date]$today"
+
+            yarn release:fluentui:pack-nightly
+
+      - task: AzureUpload@2
+        displayName: '[Nightly] Upload tarballs'
+        condition: and(succeeded(), eq(variables.publishOfficial, false))
+        inputs:
+          SourcePath: fluentui-nightly
+          ConnectedServiceNameARM: 4f791194-9458-4a18-9f9d-6a77ddb74b8c
+          StorageAccountRM: fluentsite
```

Comment:
Is it the same as `azureSubscription`? If not, why?

Reply:
that's a good point, how did you get these params, I see this in the GUI:

![image](https://user-images.githubusercontent.com/20744592/107282112-781d9800-6a52-11eb-9399-03fbf0eea3a7.png)

Reply:
https://github.com/microsoft/fluentui/blob/8f6247d89836bc8577ff618a7593fc455b016e42/azure-pipelines.yml#L153-L161

I've seen in other `AzureUpload` tasks

Reply:
No , I mean the GUI also uses `azureSubscription` so I'm not sure where these params are from

Reply:
yeah I got this from gui generated yml. Let me try to make it same as the other pipelines we have and see if it still works